c/misra/test/rules/DIR-4-9/test.c

```diff
@@ -0,0 +1,30 @@
+#define MACRO(OP, L, R) ((L)OP(R))   // COMPLIANT
+#define MACRO2(L, R) (L + R)         // COMPLIANT
+#define MACRO3(L, R) (L " " R " " L) // COMPLIANT
+#define MACRO4(L)                                                              \
+  (L" "                                                                        \
```

Comment:
The `L` prefix is a bit unfortunate because it creates a wide character string instead of using the `L` parameter. So this test case is not doing what we hope it is doing.

I noticed this because the results of `MACRO2(1,1)` should have constant resulting in an expression when we call `getAnExpandedArgument`, however the result was empty indicating that the macro parameter isn't used.

Reply:
ah, another good catch! fixed